app/index.js

```diff
@@ -1,11 +1,12 @@
-'use strict';
-const App = require('app');
-const Path = require('path');
-const BrowserWindow = require('browser-window');
-const Tray = require('tray');
+"use strict";
+const App = require("app");
+const Path = require("path");
+const BrowserWindow = require("browser-window");
+const Tray = require("tray");
```

Comment:
So I've been doing a bunch of Java programming where
`'` is strictly for `char` literals and `"` is strictly for String literals
In JS they're interchangeable so I thought to adhere to Java style but if Google says otherwise, I don't mind.